configure.py

```diff
@@ -0,0 +1,63 @@
+import os, platform, subprocess
```

Comment:
configure.py is probably not the best name.  People may assume it configures the build.

Perhaps check_build_requirements.py?

Reply:
That would be confusing, maybe check.py

Reply:
Totally fine to rename it to configure if it plays that role in the future, that makes sense.

"check" is okay, a bit generic (what is it checking?).  Currently there is nothing in the file that says what it is for, so all people have is the name.  A good description in the file (argparse and --help?  Or a nice comment) would help.